docs/src/main/sphinx/release/release-398.md

```diff
@@ -0,0 +1,64 @@
+# Release 398 (28 Sep 2022)
+
+## General
+
+* Add Hudi connector. ({issue}`10228`)
+* Add metrics for the execution time of filters and projections to `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`14135`)
+* Show local cost estimates when using `EXPLAIN`. ({issue}`14268`)
+* Fix timeouts happening too early because of improper handling of the
+  `node-scheduler.allowed-no-matching-node-period` configuration property. ({issue}`14256`)
+* Fix query failure for `MERGE` queries when `task_writer_count` is greater
+  than one. ({issue}`14306`)
+
+## CLI
+
+* Add display for the results of `EXPLAIN ANALYZE`. ({issue}`13907`)
```

Comment:
Not sure what that means.. did it not display anything in the past?

Reply:
Yeah, I don't think it did at all.

Reply:
So is that a bug fix then @martint ?